scripts/compare_benchmarks.py

```diff
@@ -84,74 +144,194 @@ def calculate_and_format_p_value(old, new):
 if old_data["context"]["benchmark_mode"] != new_data["context"]["benchmark_mode"]:
     exit("Benchmark runs with different modes (ordered/shuffled) are not comparable")
 
-diffs = []
+diffs_throughput = []
+total_runtime_old = 0
+total_runtime_new = 0
+
+add_note_for_capped_runs = False  # Flag set when max query runs was set for benchmark run
+add_note_for_insufficient_pvalue_runs = False  # Flag set when runs was insufficient for p-value calculation
 
+
+# Create table header:
+# $latency and $thrghpt (abbreviated to keep the column at a max width of 8 chars) will later be replaced with a title
+# spanning two columns
 table_data = []
-table_data.append(["Benchmark", "prev. iter/s", "runs", "new iter/s", "runs", "change [%]", "p-value"])
+table_data.append(["Item", "$latency", "", "Change", "$thrghpt", "", "Change", "p-value"])
+table_data.append(["", "old", "new", "", "old", "new", "", ""])
 
 for old, new in zip(old_data["benchmarks"], new_data["benchmarks"]):
     name = old["name"]
     if old["name"] != new["name"]:
         name += " -> " + new["name"]
 
-    if float(old["items_per_second"]) > 0.0:
-        diff = float(new["items_per_second"]) / float(old["items_per_second"])
-        diffs.append(diff)
+    # Create numpy arrays for old/new successfull/unsuccessful runs from benchmark dictionary
```

Comment:
typo

Reply:
If you mean "sucessfull", fixed. I will never learn that.